components/econet/econet.cpp

```diff
@@ -440,12 +440,14 @@ void Econet::request_strings_() {
     if (loop_now_ - last_read_request_ < min_delay_between_read_requests_) {
       return;
     }
-    for (int request_mod = 0; request_mod < request_mods_; request_mod++) {
-      if ((loop_now_ - request_mod_last_requested_[request_mod]) >= request_mod_update_interval_millis_[request_mod]) {
-        std::copy(request_datapoint_ids_[request_mod].begin(), request_datapoint_ids_[request_mod].end(),
+    std::set<uint8_t>::iterator request_mod;
```

Comment:
Do you really need this? Can't you just have auto below?

Reply:
auto worked just fine.